common/tests/unittests/test_path_converter.py

```diff
@@ -0,0 +1,41 @@
+from unittest.mock import patch
```

Comment:
We should add a unit test for the absolute path example both from linux and windows

https://learn.microsoft.com/en-us/dotnet/standard/io/file-path-formats

For example, on windows it could be something like:

"C:\Documents\foo\"